BrowserCache_Environment_Nginx.php

```diff
@@ -302,19 +296,19 @@ private function generate_section( &$rules, $mime_types, $section ) {
 				$this->c, $section );
 			$rules .= '    ' . implode( "\n    ", $subrules ) . "\n";
 
-			if ( !$this->c->get_boolean( 'browsercache.no404wp' ) ) {
-				$wp_uri = network_home_url( '', 'relative' );
-				$wp_uri = rtrim( $wp_uri, '/' );
+			if ( $this->c->get_boolean( 'browsercache.no404wp' ) ) {
```

Comment:
It looks like this condition was negated.  If `browsercache.no404wp` is true, then we should not be trying to rewrite the request through WordPress's `index.php` file.

Reply:
I believe the original condition was actually incorrect as the generate_section is called for each mime type. If this condition were left as a !condition it wouldn't apply the "no 404" functionality. I will retest and confirm this